src/main/resources/assets/tconstruct/book/materials_and_you/en_us/tools/broad_sword.json

```diff
@@ -2,18 +2,14 @@
   "tool": "tconstruct:broad_sword",
   "text": [
     {
-      "text": "The Broadsword is a versatile weapon. It has average damage, allows to knock back enemies with a sweep attack all while keeping your other hand free."
+      "text": "The Broadsword is a versatile weapon. It has average damage, and allows you to knock back your enemies with a sweep attack, all while keeping your other hand free."
```

Comment:
I want to emphasize it knocks back multiple enemies, currently reads as if it has more knockback in general. Probably as simple as saying "allows you to knock back multiple enemies at once with a sweep attack"